packages/react-composites/tests/browser/callwithchat/hermetic/Chat.test.ts

```diff
@@ -42,11 +42,11 @@ test.describe('CallWithChat Composite CallWithChat Page Tests', () => {
     await waitForMessageSeen(page);
 
     // Start typing a message
-    await startTypeMessageFromHiddenChatComposite(page, chatRemoteParticipant, 'I agree!');
+    await typeMessageFromHiddenChatComposite(page, chatRemoteParticipant, 'I agree!');
     // Wait for indicator to show up
     await waitForAndHideTypingIndicator(page, APP_UNDER_TEST_ROOT_SELECTOR);
     // Stop typing and send a message
-    await stopTypingAndSendMessageFromHiddenChatComposite(page, chatRemoteParticipant);
+    await typeAndSendMessageFromHiddenChatComposite(page, chatRemoteParticipant, 'I agree!');
```

Comment:
Looks like we are typing 'I agree' for twice and I guess this is changing the test behaivor?

Reply:
The logic for how the hidden composite is hidden has changed. We now no longer render it entirely whereas before we just marked visibility as hidden. This is because react is much better now and hooks like triggering read receipts get triggered when the composite is hidden now. Because we unmount anything typed and not sent it lost, so I've updated the functions to by type* and typeAndSend*

Reply:
Could be a work item not to use hidden composite but adapter/clients